graylog2-web-interface/src/components/events/events/BulkActions.tsx

```diff
@@ -40,8 +42,9 @@ const BulkActions = ({ selectedEntitiesData }: Props) => {
   const history = useHistory();
   const onReplaySearchClick = useCallback(() => {
     const eventIds = events.map((event) => event.id);
+    sendEventActionTelemetry('REPLAY_SEARCH', true);
```

Comment:
I think it would be helpful for us to see how many events were included when triggering this action.